### ISSUE TITLE
Resolve composer installation problem with low deps

### ISSUE THREAD

**Issue by ArturMoczulski** [rollbar/rollbar-php#284]
`composer update --prefer-dist --prefer-lowest --prefer-stable` results in 
```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - rollbar/rollbar v1.3.0 requires rollbar/rollbar-js v2.1.0 -> no matching package found.
    - rollbar/rollbar v1.3.1 requires psr/log ~1.0.2 -> satisfiable by psr/log[1.0.2].
    - rollbar/rollbar v1.3.2 requires psr/log ~1.0.2 -> satisfiable by psr/log[1.0.2].
    - rollbar/rollbar v1.3.3 requires psr/log ~1.0.2 -> satisfiable by psr/log[1.0.2].
    - Conclusion: don't install psr/log 1.0.2
    - Installation request for rollbar/rollbar ^1.3 -> satisfiable by rollbar/rollbar[v1.3.0, v1.3.1, v1.3.2, v1.3.3].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

This is causing https://travis-ci.org/Seldaek/monolog/jobs/296984544 build to fail. Probably the same issues would happen in any other project when running `--prefer-dist --prefer-lowest --prefer-stable` options.

**Comment by ArturMoczulski** [comment]
The SDK installed with `--prefer-dist --prefer-lowest --prefer-stable` depends on `Seldaek/monolog:1.0.0` which breaks the `Rollbar\ReadMeTest::testMonolog`. We need to fix this first and then check if Rollbar PHP is able to run on `psr/log:1.0.1` which is the preferred version by the Monolog library when also installed with `composer update --prefer-dist --prefer-lowest --prefer-stable`.

1. We need to fix `Rollbar\ReadMeTest::testMonolog` when on `--prefer-dist --prefer-lowest --prefer-stable`.
2. Check that our test suite is passing on `psr/log:1.0.1`.
3. If so, change our `psr/log` requirement to version constraint `^1.0.1` and merge is into the tag.
4. https://github.com/Seldaek/monolog/pull/1042 build should be fixed on low deps.

**Comment by ArturMoczulski** [comment]
This is fixed in tag `1.3.4`
